environment.yml

```diff
@@ -2,6 +2,7 @@ name: utilsforecast
 channels:
   - conda-forge
 dependencies:
+  - python=3.10
```

Comment:
why?

Reply:
It states it in the contributing.md. Without this, it creates an env by default using 3.13 when the command is run

![image](https://github.com/user-attachments/assets/63bc4b8f-3952-410f-8acb-5fdb97f330a0)

Reply:
I am switched the instructions to use `uv` instead of `conda` so we can move closer to the current standards. Hope that is fine and will address this comment.